4-fleetscope/modules/env_baseline/main.tf

```diff
@@ -14,4 +14,22 @@
  * limitations under the License.
  */
 
-#TODO Add GKE cluster definition
+resource "google_gke_hub_scope" "fleet-scope" {
+  scope_id = var.scope_id
+  project = var.project_id
+}
+
+resource "google_gke_hub_namespace" "fleet-ns" {
+  scope_namespace_id = var.namespace_id
```

Comment:
`terraform fmt` can be used to automatically fix the field indentation.

Reply:
from the root of the repo, you can run `terraform fmt -recursive` to take care of all spacing issues at once @yliaog

Reply:
done, thanks